datalad/crawler/nodes/misc.py

```diff
@@ -12,8 +12,10 @@
 import inspect
 import os
 import re
+import stat
 
-from os.path import curdir, join as opj, split as ops
+from stat import *
```

Comment:
in general `import *` is discouraged, please list each imported entity explicitly